services/emailer/src/namex_emailer/email_templates/NR-BEFORE-EXPIRY-IA.html

```diff
@@ -0,0 +1,50 @@
+# Your Name Request is Expiring Soon
+
+Your Name Request {{ nr_number }} for **{{ legal_name }}** is due to expire on {{ expiration_date }}.
+
+If your name request expires:
+
+* The approved name is no longer reserved for business registration and becomes available to the public
+* You will need to submit and pay for a new name request
+
+Finish registering a business with this name before it expires. If you are not ready to register a business yet , you can renew this name request to extend the expiry date.
+
+---
+
+# Option 1: Finish Registering a Business
+
+Follow these steps to register your business using the approved name:
+
+1. Visit [BC Registries and Online Services]({{ magic_link }})
```

Comment:
Fixed

Reply:
It looks like this still has the old steps instead of the new modernized steps.

Confirm with Jacqueline whether any other changes are needed to this template.

Reply:
Updated